src/features/singleRight/delegate/ChooseRightsPage/ChooseRightsPage.tsx

```diff
@@ -66,6 +69,7 @@ export const ChooseRightsPage = () => {
   const processedDelegationsRatio = (): number =>
     Math.round((processedDelegations.length / delegationCount) * 100);
   const buttonRef = useRef<HTMLButtonElement | null>(null);
+  const [clearAccessCashe] = useClearAccessCasheMutation();
```

Comment:
I don't understand why this is a mutation, we don't use any data from that call? So a simple axios call would do. It's not critical for release but anyways

Reply:
It's a mutation because it's a PUT call (and it technically does change something on backend, namely the cache)
I put it in the SingleRightsApi as an rtkQuery because it's related to the rest of the SingleRights feature and part of the BFF's API's so this way it adheres to our already established pattern.
Not to mention that it's honestly just as easy to add it to the rtkQuery as it would have been to use an axios call (with the added benefit of abstracting the actual network call away from this file) 😊